bloom-housing/bloom
# feat: add farmworkerHousing translations

This PR addresses [#(910)](https://github.com/metrotranscom/doorway/issues/910)


## Description

This adds the farmworkerHousing translations needed for Doorway. It also corrects the plural versions of a few reservedCommunityTypeDescriptions and the traditional Chinese translations for some cases.
Added examples for the createNewReservedCommunityType endpoint to make clear what is needed in the fields.

## How Can This Be Tested/Reviewed?

1. Reseed the backend and start it up
2. Log in as a admin user (either through the partner UI or directly in the swagger doc)
3. In the swagger doc [http://localhost:3100/api#/scriptRunner/createNewReservedCommunityType](http://localhost:3100/api#/scriptRunner/createNewReservedCommunityType) run the script 3 times using the following data:
{
id : jurisdiction id (need to get it from the db)
name: "farmworkerHousing"
description: "Employee of the school district"
}
{
id : jurisdiction id (need to get it from the db)
name: "schoolEmployee"
description: "Farmworkers and Their Families"
}
{
id : jurisdiction id (need to get it from the db)
name: "housingVoucher"
description: "HCV/Section 8 Voucher Holders"
}
4. The new community type should be in the database and also selectable in the partner UI
5. Test each of the above by adding them to a listing in the assigned jurisdiction and toggling the language
6. Additionally test in the same way the Veteran reservedCommunityType
## Author Checklist:


## Review Process:

- Read and understand the issue
- Ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Either (1) explicitly ask a clarifying question, (2) request changes, or (3) approve the PR, even if there are very small remaining changes, if you don't need to re-review after the updates
